application/event_processor/change_event_validation.py

```diff
@@ -21,8 +21,7 @@ def validate_event(event: Dict[str, Any]) -> None:
     try:
         validate(event=event, schema=INPUT_SCHEMA)
     except SchemaValidationError as exception:
-        logger.exception(f"Input schema validation error|{str(exception)}", extra={"event": event})
-        raise ValidationException("Change Event malformed, validation failed")
+        raise ValidationException(str(exception))
```

Comment:
why cast exception string? ValidationException accepts argument of type Exception

Reply:
Good spot, I was moving it from the above log and missed that.